pylint/testutils/utils.py

```diff
@@ -355,25 +267,14 @@ def to_csv(self):
 _EXPECTED_RE = re.compile(
     r"\s*#\s*(?:(?P<line>[+-]?[0-9]+):)?"
     r"(?:(?P<op>[><=]+) *(?P<version>[0-9.]+):)?"
-    r"\s*\[(?P<msgs>%(msg)s(?:,\s*%(msg)s)*)\]" % _MESSAGE
+    r"\s*\[(?P<msgs>%(msg)s(?:,\s*%(msg)s)*)]" % _MESSAGE
```

Comment:
Does it worth it? I feel clearer to keep the escaping character before both [ and ].

Reply:
Resolving because there is the same issue discussed here : https://github.com/PyCQA/pylint/pull/3943#discussion_r531431346